recipes/_configure_server.rb

```diff
@@ -105,7 +105,7 @@
 
 license_group =
   case node['splunk']['node_type']
-  when :license_server, :cluster_master, :cluster_slave
+  when :cluster_master, :cluster_slave, :heavy_forwarder, :license_server
```

Comment:
I think not enabling indexing out of the box is fine, as the purpose of this recipe is primarily to forward data... I'm a bit more disappointed to find out that LDAPAuth is in fact an Enterprise feature, and not part of the Forwarder license. Initial use cases this might be ok though.